nfpm/zrok-share.env

```diff
@@ -86,3 +86,7 @@ ZROK_SHARE_OPTS=""
 # set if self-hosting zrok and not using only the default frontend name 'public'; must be a space-separated list
 # WARNING: changes take effect the next time the frontend URL is reserved
 #ZROK_FRONTENDS="public"
+
+# you MAY set to change the frontend mode: temp-public (default), temp-private, reserved-public, reserved-private
```

Comment:
As mentioned in the other comment, let's preserve default `reserved-public` and make the temp options less visible since they're rarely useful when not running zrok interactively.

Reply:
:+1: Agreed.